src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -116,9 +116,16 @@ public static function register($mode = 0)
             }
 
             if (isset($trace[$i]['object']) || isset($trace[$i]['class'])) {
-                $class = isset($trace[$i]['object']) ? get_class($trace[$i]['object']) : $trace[$i]['class'];
-                $method = $trace[$i]['function'];
-                $Test = $UtilPrefix.'Test';
+                if (isset($trace[$i]['class']) && 'Symfony\Bridge\PhpUnit\SymfonyTestsListener' === $trace[$i]['class']) {
+                    $parsedMsg = unserialize($msg);
+                    $msg = $parsedMsg['deprecation'];
+                    $class = $parsedMsg['class'];
+                    $method = $parsedMsg['method'];
+                } else {
+                    $class = isset($trace[$i]['object']) ? get_class($trace[$i]['object']) : $trace[$i]['class'];
+                    $method = $trace[$i]['function'];
+                    $Test = $UtilPrefix.'Test';
```

Comment:
$Test is not assigned if the class is Symfony\Bridge\PhpUnit\SymfonyTestsListener. This might cause problems in the later if

Reply:
indeed, fixed